app/views/posts/show.html.erb

```diff
@@ -1,6 +1,12 @@
-<h1>List of current user's  posts</h1>
-<ul>
-  <li>post 1</li>
-  <li>post 2</li>
-  <li>post 3</li>
-</ul>
\ No newline at end of file
+<div>
+  <div>
+    <h4>Post #<%= @post.id %> by <%= @post.author.name %></h4>
+    <p>Comments: <%= @post.comments.count %>, Likes: <%= @post.likes.count %></p>
+    <p><%= @post.text %></p>
+  </div>
+  <div>
+    <% @comments.each do |comment| %>
+      <p><%=comment.user.name %>: <%= comment.text %></p>
+    <% end %>
+  </div>
+</div>
```

Comment:
| @Alibaba2023 Post Show | Microverse Post Show |
|-|-|
| ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/6dcf5560-9f8a-4ad6-a8a6-7b895d468ac3) | ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/e5496462-908f-48ce-bfbb-209ea97c575f) |

- [ ] It would be great if you follow the given wireframe to follow the project requirements to make your design look professional 👍